ckanext/security/cache/login.py

```diff
@@ -51,24 +47,32 @@ def increment(self):
         # the user would be locked out for another `login_lock_timeout` minutes
         # whenever he/she tries to login again.
         if self.count < self.login_max_count + 1:
-            value.update({self.user_name: "%s:%s" % (self.count + 1, self.request_time)})
```

Comment:
Previously the user_name attribute was storing a representation of the failed login count and last failed login timestamp. I don't understand why. I've moved to having explicit attributes to store these values instead.